index.js

```diff
@@ -26,12 +25,12 @@ function explorer(loopbackApplication, options) {
     apiInfo: loopbackApplication.get('apiInfo') || {}
   });
 
-  var app = express();
+  var loopback = loopbackApplication.loopback;
```

Comment:
This is a breaking change, because `app.loopback` was added in `v1.9.0`, thus the explorer won't work anymore with older loopback versions. We should release this change as a major version.